server/src/weaverbird/backends/pypika_translator/translators/base.py

```diff
@@ -305,6 +315,38 @@ def get_sql(self, **kwargs: Any) -> str:
         # without actually executing the query
         return custom_query, StepTable(columns=["*"])
 
+    def cumsum(
```

Comment:
Instead of having a broken step for Snowflake and all others, perhapse you could: 
1. Override the method for Snowflake Translator
2. Raise unimplemented for others.

Reply:
i will rather comment this function for now, since, it's not working on snowflake's edge cases, thanks to pointing this out 🙏🏾